src/rust-crypto/backup.ts

```diff
@@ -164,9 +165,10 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
         // We force a check to ensure to have the latest version. We also want to check that the backup is trusted
         // as we don't want to store the secret if the backup is not trusted, and eventually import megolm keys later from an untrusted backup.
```

Comment:
this comment seems to be out of date.

```suggestion
        // We force a check to ensure to have the latest version.
```